robocorp-code/vscode-client/src/conversion.ts

```diff
@@ -374,7 +378,13 @@ export async function convertAndSaveResults(
                     outputDirsWrittenTo.add(result.outputDir);
                     if (files && files.length > 0) {
                         for (const f of files) {
-                            tasks.push(handleOutputFile(join(result.outputDir, f.filename), f.content));
+                            tasks.push(
+                                handleOutputFile(
+                                    join(result.outputDir, f.filename),
+                                    f.content,
+                                    f.encoding as BufferEncoding
+                                )
+                            );
                         }
                     }
                     const images = okResult?.images;
```

Comment:
A question: should the `handleOutputFile` for the images have `binary` by default?

(do we even use those?)